### DISCUSSION TITLE
Repeated @font-face in Global styles

### DISCUSSION THREAD

**Discussion by andy-zhou** [chakra-ui/chakra-ui#2651] [Q&A] [answered]
Is there any way to add repeated `@font-face` CSS at-rules in global styles? I've tried both the `css` tag and [object-style arrays](https://emotion.sh/docs/object-styles#arrays), but neither seem to work.

**Comment by with-heart** [comment, marked as answer]
Hi @andy-zhou! The `styles` prop of `Global` from `@emotion/react` allows you to insert raw css where you can add repeat `@font-face` declarations: https://codesandbox.io/s/bold-bouman-2sp5v?file=/src/App.tsx
